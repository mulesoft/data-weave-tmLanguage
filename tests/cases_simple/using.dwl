fun printPassCapable(item) =
  using(msg = aa)
    item

fun printPassCapable(item) =
  using(msg = println(1 map 3))
    item map 3