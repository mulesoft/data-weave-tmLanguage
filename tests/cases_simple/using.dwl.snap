>fun printPassCapable(item) =
#^^^ source.data-weave storage.type.dw
#   ^ source.data-weave
#    ^^^^^^^^^^^^^^^^ source.data-weave entity.name.variable.dw
#                    ^ source.data-weave
#                     ^^^^ source.data-weave entity.name.variable.dw
#                         ^^ source.data-weave
#                           ^ source.data-weave keyword.operator.assignment.dw
>  using(msg = aa)
#^^ source.data-weave
#  ^^^^^ source.data-weave keyword.control.dw
#       ^ source.data-weave
#        ^^^ source.data-weave entity.name.variable.dw
#           ^ source.data-weave
#            ^ source.data-weave keyword.operator.assignment.dw
#             ^ source.data-weave
#              ^^ source.data-weave entity.name.variable.dw
#                ^^ source.data-weave
>    item
#^^^^ source.data-weave
#    ^^^^ source.data-weave entity.name.variable.dw
>
>fun printPassCapable(item) =
#^^^ source.data-weave storage.type.dw
#   ^ source.data-weave
#    ^^^^^^^^^^^^^^^^ source.data-weave entity.name.variable.dw
#                    ^ source.data-weave
#                     ^^^^ source.data-weave entity.name.variable.dw
#                         ^^ source.data-weave
#                           ^ source.data-weave keyword.operator.assignment.dw
>  using(msg = println(1 map 3))
#^^ source.data-weave
#  ^^^^^ source.data-weave keyword.control.dw
#       ^ source.data-weave
#        ^^^ source.data-weave entity.name.variable.dw
#           ^ source.data-weave
#            ^ source.data-weave keyword.operator.assignment.dw
#             ^ source.data-weave
#              ^^^^^^^ source.data-weave entity.name.variable.dw
#                     ^ source.data-weave
#                      ^ source.data-weave constant.numeric.dw
#                       ^ source.data-weave
#                        ^^^ source.data-weave entity.name.variable.dw
#                           ^ source.data-weave
#                            ^ source.data-weave constant.numeric.dw
#                             ^^^ source.data-weave
>    item map 3
#^^^^ source.data-weave
#    ^^^^ source.data-weave entity.name.variable.dw
#        ^ source.data-weave
#         ^^^ source.data-weave entity.name.variable.dw
#            ^ source.data-weave
#             ^ source.data-weave constant.numeric.dw